eidpsam/src/test/java/uk/ac/ucl/eidp/auth/jaspic/HttpHeaderAuthModuleTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.ac.ucl.eidp.auth.jaspic;
+
+import java.security.Principal;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'org\.testng\.annotations\.BeforeMethod' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.